SofaKernel/modules/SofaBaseMechanics/MechanicalObject.h

```diff
@@ -422,8 +417,11 @@ protected :
 
     /// @}
 
-    sofa::core::objectmodel::DataFileName filename;
-    Data< bool> ignoreLoader;
+    //int vsize; ///< Number of elements to allocate in vectors
+    Data< int > d_size;
```

Comment:
Why not but... why make it a Data?
What does happen if this Data value changes?
Note that a DataTracker could be used to automatically resize the mstate as soon as this Data is changed.